### DISCUSSION TITLE
There does not seem to be any mention of isFetching in the v4 documentation.

### DISCUSSION THREAD

**Discussion by a-yoshizawa-mediba** [TanStack/query#5884] [General]
The [v3 documentation](https://tanstack.com/query/v3/docs/react/guides/queries) mentions isFetching, but the [v4 documentation](https://tanstack.com/query/v4/docs/react/guides/queries) does not seem to have it. Should I mention this in the issue?

**Comment by TkDodo** [comment]
`isFetching` is a flag derived from the `fetchStatus` (`fetchStatus === 'fetching'`). Feel free to open a PR to the docs.
